tests/test_utils.py

```diff
@@ -166,7 +166,7 @@ def _test_with_indices(self, indices: torch.Tensor) -> None:
         assert emb.shape == (num_ind, 1, self.embedding_dim)
 
         # check values
-        exp = torch.stack([self.embedding(i) for i in indices], dim=0).view(num_ind, 1, self.embedding_dim)
+        exp = torch.stack([self.embedding(i.view(1)) for i in indices], dim=0).view(num_ind, 1, self.embedding_dim)
```

Comment:
what is this 1 a measure of? the length of the shape tuple?

Reply:
The syntax is `view(*shape)` -> the result shape is `(1,)` (a 1-d tensor with one element) instead of `tuple()` (a scalar tensor).